### ISSUE TITLE
SET datatype not handled correctly

### ISSUE THREAD

**Issue by seb303** [doctrine/sql-formatter#94]
It should not start a new line before and after the "SET" like it does here:
![image](https://github.com/doctrine/sql-formatter/assets/467942/44b43e24-ad6b-4cbf-86f4-64d8193aa284)
![image](https://github.com/doctrine/sql-formatter/assets/467942/ae21528e-fa7c-4812-ba4f-6870bd14a5c3)
Here's the raw query for the 2nd example:
`ALTER TABLE o2_test ADD COLUMN enum_test2 ENUM('va1','val2','val3') AFTER set_test, ADD COLUMN set_test2 SET('va1','val2','val3') AFTER enum_test2`

**Comment by Menelion** [comment]
It seems to have problem with the `SET` keyword in all contexts. I'll create a new issue about `CHARACTER SET` which it splits for some reason.
